test/test-execute/exec-systemcallerrornumber-name.service

```diff
@@ -2,7 +2,7 @@
 Description=Test for SystemCallErrorNumber
 
 [Service]
-ExecStart=/bin/sh -x -c 'uname -a'
+ExecStart=/bin/python3 -c 'import os\ntry: os.uname()\nexcept Exception as e: exit(e.errno)'
```

Comment:
On Ubuntu Xenial there is no `/bin/python3`, so `/usr/bin/python3` has to be used instead to make the test work there. I would prefer `/bin/sh -c` so as to avoid guessing where something is as happened recently  in https://github.com/systemd/systemd/pull/7276 and in https://github.com/systemd/systemd/pull/7062#discussion_r144175704, though.

Reply:
Thanks, fixed in #7306.